fireblocks_sdk/api_types.py

```diff
@@ -283,6 +283,39 @@ def __init__(self, total_amount_larger_than=None, asset_id=None, order_by=None,
         self.after = after
 
 
+class GetSmartTransferFilters(object):
+    """ Optional filters to apply for request
+    Args
+        q (string, optional):  Search query string - either ticketId, asset or network name
+        statuses (DRAFT/PENDING_APPROVAL/OPEN/IN_SETTLEMENT/FULFILLED/EXPIRED/CANCELED, optional): array of ticket statuses
+        network_id (string, optional): networkId used in ticket
+        created_by_me (bool, optional): created by me flag
+        expires_after (string, optional): Lower bound of search range
+        expires_before (string, optional): Upper bound of search range
+        ticket_type (ASYNC/ATOMIC, optional): type of ticket
+        external_ref_id (string, optional): external ref id
+        after (string, optional): cursor string received from previous request
+        limit (number, optional): Results page size
+
+    Constraints
+        - You should only insert 'before' or 'after' (or none of them), but not both
+    """
+
+    def __init__(self, q: str = None, statuses: str = None, network_id: str = None, created_by_me: bool = None,
+                 expires_after: str = None, expires_before: str = None, ticket_type: str = None,
+                 external_ref_id: str = None, after: str = None, limit: str = None):
```

Comment:
when you do `= None` the type can be None and not only the str/bool declared. The type should be wrapped with `Optional[]`

Reply:
Please update all functions (not only this one)

Reply:
done. Can you please re-review?